src/Mutation/PostObjectCreate.php

```diff
@@ -296,13 +296,6 @@ public static function mutate_and_get_payload( $post_type_object, $mutation_name
 				}
 			}
 
-			/**
-			 * If the $post_id is empty, we should throw an exception
-			 */
-			if ( empty( $post_id ) ) {
```

Comment:
should be added back (or refactored to match the other tests for a return value of `0`): https://developer.wordpress.org/reference/functions/wp_insert_post/